client/src/components/Room.tsx

```diff
@@ -9,6 +10,8 @@ export const Room: React.FunctionComponent = () => {
     const { id } = useParams()
     const { T } = useTranslationStore()
     const { ws, peer, stream, peers } = useRoomStore()
+    const [activeCamera, setActiveCamera] = useState(true)
+    const [activeMicrophone, setActiveMicrophone] = useState(true)
```

Comment:
I would make a store with active streams and information about is camera on and microphone